### DISCUSSION TITLE
blog/disable-github-copilot-markdown-vscode

### DISCUSSION THREAD

**Discussion by giscus** [AshikNesin/nesinio-comments#15] [Announcements]
# blog/disable-github-copilot-markdown-vscode

Discover how to disable markdown GitHub Copilot suggestions/distractions in markdown files

https://nesin.io/blog/disable-github-copilot-markdown-vscode

**Comment by PaulTopping** [comment]
Looks great! Anyone know how to do this in Visual Studio 2022?
